usr/iface.c

```diff
@@ -44,6 +44,8 @@
 #include "iscsi_err.h"
 #include "iscsi_netlink.h"
 
+#define _unwrap(x) strlen(x) ? x : UNKNOWN_VALUE
+
```

Comment:
Is rust rubbing off on you? :-)

Reply:
Any possibility of x being NULL?

Reply:
Yeah. That's rust.
The pointer will not be NULL, but I still change it to:
`#define _unwrap(x) (x && strlen(x)) ? x : UNKNOWN_VALUE`